src/main/java/com/internet/shop/model/Order.java

```diff
@@ -13,6 +13,15 @@ public Order(Long userId) {
         this.userId = userId;
     }
 
+    public Order() {
+
+    }
```

Comment:
```suggestion
    public Order() {
    }
```

Reply:
Reconsider creating this constructor after replacing 

**_new Order();_** with the _**null**_

Same in similar cases with other models

Reply:
Of course, they are unnecessary.

Reply:
But they will be used in the future, I will keep them.